turbinia/api/cli/README.md

```diff
@@ -1 +1,164 @@
-# turbinia-client
+# Turbinia API CLI tool (turbinia-client)
+
+## Summary
+Turbinia-client is a command-line tool that provides an easy-to-use user interface for Turbinia's API server. It allows a user to perform Turbinia operations such as creating new Turbinia processing requests, getting status of existing requests and tasks, and downloading requests output and logs.
+
+## Getting started
+To get started with the Turbinia API CLI tool you will need to install it on your system. To install the tool, follow the instructions below.
+
+### Installation
+You can install the latest version of the tool via PyPi.
+```
+pip install turbinia-client
+```
+The package will install the ```turbinia-api-lib``` library as a dependency. More information on how to use the ```turbinia-api-lib``` Python library can be found [here](https://github.com/google/turbinia/master/turbinia/api/client).
+
+### Configuring the client
+The command-line tool uses a JSON configuration file. By default, the client will search for a ```.turbinia_api_config.json``` file within the user's home directory, or a path specified in the ``TURBINIA_API_CONFIG_PATH``` environment variable.
+
+Support for multiple Turbinia environments is possible in the configuration file. In the example below, there are two Turbinia environments, ```default``` and ```development```.
```

Comment:
Should we reference anything around configuring authentication? Even though each Oauth2 Provider will be different maybe can at least mention, something along the line of *Turbinia utilizes Oauth2 for authentication and an example set of client credentials  can be obtained here.

Reply:
This is mentioned in the authentication section in the new docs in #1232